# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLFramebuffer.h
# Commit: 0b191b707750
# Full Hash: 0b191b707750e040289033359306e4966c1bf062
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - WebXR compositing fixes. r=imanol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78117
# ==============================================================================

diff -r 16c5dd2cfd60 -r 0b191b707750 dom/canvas/WebGLFramebuffer.h
--- a/dom/canvas/WebGLFramebuffer.h	Mon Jun 15 18:25:55 2020 +0000
+++ b/dom/canvas/WebGLFramebuffer.h	Mon Jun 15 18:25:58 2020 +0000
@@ -10,6 +10,7 @@
 
 #include "mozilla/WeakPtr.h"
 
+#include "GLScreenBuffer.h"
 #include "WebGLObjectModel.h"
 #include "WebGLStrongTypes.h"
 #include "WebGLTexture.h"
@@ -138,6 +139,7 @@
   const GLuint mGLName;
   bool mHasBeenBound = false;
   const UniquePtr<gl::MozFramebuffer> mOpaque;
+  gl::SwapChain mOpaqueSwapChain;
   bool mInOpaqueRAF = false;
 
  private: